packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -430,7 +226,7 @@ export const WorkspaceSettings = () => {
               <Button
                 autoWidth
                 variant="link"
-                disabled={loading}
+                disabled={teamInfoLoading}
```

Comment:
Do we really need to disable this button here? I've tried updating the team but it never seems to resolve and is stuck in the loading state. It's either in a loop [related to this `useEffect`](https://github.com/codesandbox/codesandbox-client/pull/7509/files#diff-2e799785ade7c78119b46da1285e889ba2c1afe1dacff2493ad214762cda2435R94-R96), or something else is going on. I think it's simpler if we keep the loading state inside the `TeamInfo` component rather than lifting it up to `WorkspaceSettings`.

Reply:
@tristandubbeld makes sense to not disabled, I tried to keep the behaviour when refactoring but since the only blocker to deleting a team is having an active subscription, we can remove the check to make things easier.